README.Rmd

```diff
@@ -23,7 +23,7 @@ knitr::opts_chunk$set(
 <!-- badges: end -->
 
 An R package used for deploying applications to the [ShinyApps](https://www.shinyapps.io/) hosted service, 
-or to [RStudio Connect](https://www.rstudio.com/products/connect/).
```

Comment:
The `README.Rmd` came to exist in https://github.com/rstudio/rsconnect/pull/380, but I don't see any explicit workflow to create `README.md` from the Rmd version.

I have typically used the "Knit" button from RStudio to recreate the Markdown file.

I would be OK ditching the Rmd in favor of the GH Markdown version.

@andrie - is there a reason to maintain both?

aside: Not sure why, but we're using `rstudioapi` in the URL above, when we should be using `rsconnect`.

Reply:
To create the `.md` from the `.Rmd` I also use the `knit` button in RStudio.  The `.Rmd` is marginally easier to maintain with the tools in the `usethis` package, e.g. to add badges.

Reply:
I got this one to knit, but still having issues with `connectwidgets`' README. Will continue any necessary discussion over in that repo :)

Reply:
@andrie - I'm inclined to ditch the separate `Rmd` and `md` files in favor of only using `README.md`. Without an explicit flow, folks forget to keep them in sync, and most folks look first for `README.md`.